README.md

```diff
@@ -83,102 +84,84 @@ You can find our official [Open Source Maintenance Policy](open_source_maintenan
-  </tr>
-  <tr>
-    <th>Frameworks</th>
-    <td>node.js</td>
-  </tr>
-    <tr>
-    <th>Link</th>
-    <td><a href="https://github.com/OperationCode/town-crier">GitHub Repo</a></td>
-  </tr>
-</table>
-
-## OperationCode Bot
+## OperationCode PyBot
 
 <table>
   <tr>
     <th>Purpose</th>
-    <td>Slack bot for dealing with slack events. Primarily used to display interactive slack messages to create a menu-like interface</td>
+    <td>Slack python bot for dealing with slack events.</td>
   </tr>
   <tr>
     <th>Languages</th>
-    <td><a href="https://github.com/OperationCode/operationcode_bot/search?l=ruby">Ruby<a/></td>
+    <td><a href="https://www.python.org/">Python</a></td>
   </tr>
   <tr>
     <th>Frameworks</th>
-    <td>Sinatra</td>
+    <td><a href="https://github.com/pyslackers/sir-bot-a-lot-2">Sir-bot-a-lot</a></td>
   </tr>
     <tr>
     <th>Link</th>
-    <td><a href="https://github.com/OperationCode/operationcode_bot">GitHub Repo</a></td>
+    <td><a href="https://github.com/OperationCode/operationcode-pybot">GitHub Repo</a></td>
   </tr>
 </table>
 
-## OperationCodePyBot
+
+## OperationCode Resources-API
 
 <table>
   <tr>
     <th>Purpose</th>
-    <td>Slack python bot for dealing with slack events. Primary use for interacting with new members. Currently in initial development stages.</td>
+    <td>Flask API for programming and cyber security learning resources.</td>
   </tr>
   <tr>
     <th>Languages</th>
-    <td><a href="https://github.com/OperationCode/operationcode-pybot/search?l=python">Python<a/></td>
+    <td><a href="https://www.python.org/">Python</a></td>
   </tr>
   <tr>
     <th>Frameworks</th>
-    <td>Sinatra</td>
+    <td><a href="http://flask.pocoo.org/">Flask</a></td>
   </tr>
     <tr>
     <th>Link</th>
-    <td><a href="https://github.com/OperationCode/operationcode-pybot">GitHub Repo</a></td>
+    <td><a href="https://github.com/OperationCode/resources_api">GitHub Repo</a></td>
   </tr>
 </table>
 
-## OperationCode Slash Commands
+## Town Crier
 
 <table>
   <tr>
     <th>Purpose</th>
-    <td>Slack bot that allows users to get information via slack's slash command UI (eg <code>/events<code>)  </td>
+    <td>Slack bot with a <code>cron</code> like interface to notify members of Operation Code opportunities  </td>
   </tr>
   <tr>
     <th>Languages</th>
-    <td><a href="https://github.com/OperationCode/operationcode_slashbot/search?l=javascript">Javascript<a/></td>
+    <td><a href="https://github.com/OperationCode/town-crier/search?l=javascript">Javascript</a></td>
   </tr>
   <tr>
     <th>Frameworks</th>
-    <td>node.js</td>
+    <td><a href="https://nodejs.org">Node.js</a></td>
   </tr>
     <tr>
     <th>Link</th>
-    <td><a href="https://github.com/OperationCode/operationcode_slash_commands ">GitHub Repo</a></td>
+    <td><a href="https://github.com/OperationCode/town-crier">GitHub Repo</a></td>
   </tr>
 </table>
 
 ## Opentroops Projects
+
```

Comment:
I don't think this is clear that while we are affiliated with them these projects are not in our organization.